travis/test_pylint

```diff
@@ -12,13 +12,48 @@ import travis_helpers
 
 from getaddons import get_modules_changed
 
+
+def get_extra_params(odoo_version):
+    '''
+    Get extra pylint params by odoo version
+    :param version: String with name of version of odoo
+    :return: List of extra pylint params
+    '''
+    params = ["--load-plugins=pylint_oca"]
+    try:
+        version_f = float(odoo_version)
+    except ValueError:
+        # master case
+        version_f = -1
+    if version_f != -1 and version_f < 8:
+        # Class name snake_case style to version < 8
+        params.append(
+            '--class-rgx='
+            '([a-z_][a-z0-9_]{2,45})|([A-Z_][a-zA-Z0-9]{2,45})$'
+        )
+        params.append(
+            '--disable='
+            'missing-readme,manifest-deprecated-key,copy-wo-api-one'
+        )
+    return params
+
+
+version = os.environ.get('VERSION', False)
+extra_params_cmd = []
+if version:
+    extra_params = get_extra_params(version)
+    for extra_param in extra_params:
+        extra_params_cmd.extend([
+        '--extra-params', extra_param])
```

Comment:
I suggest a different design, avoiding to dynamically add lint parameters inside the code:
Instead keep different config files for different Odoo versions (in different directories?).
The code should just detect the target version and then use the appropriate config file for each case.
This is a better separation of concerns.